go.mod

```diff
@@ -12,8 +13,12 @@ require (
 
 require (
 	github.com/BurntSushi/toml v1.3.2 // indirect
+	github.com/Microsoft/go-winio v0.4.14 // indirect
 	github.com/andybalholm/brotli v1.0.5 // indirect
 	github.com/cloudflare/circl v1.3.3 // indirect
+	github.com/docker/distribution v2.8.2+incompatible // indirect
```

Comment:
What incompatible means here?

Reply:
"if the imported v2+ package has a valid [semver](https://semver.org/) tag, it will be recorded with a +incompatible suffix" https://go.dev/wiki/Modules#can-a-module-consume-a-v2-package-that-has-not-opted-into-modules-what-does-incompatible-mean